scripts/populate_lambda_ssm_parameters.py

```diff
@@ -52,6 +52,11 @@ def set_deployed_lambda_environment(name, env: dict):
         }
     )
 
+def get_cache_criteria():
+    with open("{}/checkout_cache_criteria.json".format(os.environ['DSS_HOME']), "r") as file:
+        temp = "".join(file.read().splitlines())
+    return json.loads(temp)
```

Comment:
Since `checkout_cache_criteria.json` is json formatted, you can just return `temp` directly.